src/nvim/ui.c

```diff
@@ -537,6 +537,8 @@ static void ui_mode_change(void)
     mode = REPLACE;
   else if (State & INSERT)
     mode = INSERT;
+  else if (State & CMDLINE)
```

Comment:
you need to change this to use braces (the entire if/else if/...) for the linter to pass.